how-tos/setup-staging-release.md

```diff
@@ -34,6 +34,8 @@ hg -R build/jamun out -r . jamun
 hg -R build/jamun push
 ```
 
+Optional: reset the staging tools repo using this <a href="../scripts/staging_merge.py">script</a> to ensure you are using the same version as production.  You'll have to update the script with the location of your key.
```

Comment:
Nit: I think the href should be pointing to `scripts/reset_tools.sh`, not the `staging_merge.py`, right?